package.json

```diff
@@ -84,7 +84,12 @@
     "style-loader": "^0.12.3",
     "url-loader": "^0.5.7",
     "webpack": "^1.8.2",
-    "react-addons-perf" : "0.14.7"
+    "react-addons-perf" : "0.14.7",
+    "eslint-plugin-jsx-a11y": "^0.6.2",
+    "jsdoc-jsx": "^0.1.0",
+    "minami": "^1.1.1",
+    "ink-docstrap" :"^1.2.1",
+    "file-loader":"^0.09"
 
```

Comment:
Please, remove the packages that we don't use at the moment. (We can add them back, when we need them.)